src/index.ts

```diff
@@ -26,6 +26,7 @@ import './components/Radio/Radio.css'
 import './components/Segment/Segment.css'
 import './components/Table/Table.css'
 import './components/Checkbox/Checkbox'
+import './components/AssetCard/AssetCard'
```

Comment:
You seem to be missing the export of the `AssetImage` component, would you mind adding it?